[2025-02-28T14:49:23.829+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-28T14:49:23.938+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.save_to_json scheduled__2025-02-28T06:00:00+00:00 [queued]>
[2025-02-28T14:49:24.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.save_to_json scheduled__2025-02-28T06:00:00+00:00 [queued]>
[2025-02-28T14:49:24.023+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-02-28T14:49:24.077+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_to_json> on 2025-02-28 06:00:00+00:00
[2025-02-28T14:49:24.103+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_etl', 'save_to_json', 'scheduled__2025-02-28T06:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpn8lbex6j']
[2025-02-28T14:49:24.114+0000] {standard_task_runner.py:91} INFO - Job 118: Subtask save_to_json
[2025-02-28T14:49:24.126+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T14:49:24.131+0000] {standard_task_runner.py:63} INFO - Started process 226 to run task
[2025-02-28T14:49:24.581+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl.save_to_json scheduled__2025-02-28T06:00:00+00:00 [running]> on host 083d33cb466d
[2025-02-28T14:49:26.526+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl' AIRFLOW_CTX_TASK_ID='save_to_json' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T06:00:00+00:00'
[2025-02-28T14:49:26.529+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-28T14:49:26.590+0000] {logging_mixin.py:188} INFO - ✅ Đã lưu 19 bản ghi vào data/2025-02/2025-02-28.jsonl
[2025-02-28T14:49:26.593+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-28T14:49:26.595+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-28T14:49:26.618+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl, task_id=save_to_json, run_id=scheduled__2025-02-28T06:00:00+00:00, execution_date=20250228T060000, start_date=20250228T144923, end_date=20250228T144926
[2025-02-28T14:49:26.669+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-28T14:49:26.690+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-28T14:49:26.695+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
